CRVResponse/src/CrvPhotonGenerator_module.cc

```diff
@@ -202,18 +210,31 @@ namespace mu2e
         {
            tableLoaded=true;
            _makeCrvPhotons.emplace_back(_makeCrvPhotons[j]);
-           std::cout<<"CRV sector "<<i<<" ("<<_CRVSectors[i]<<") uses "<<_makeCrvPhotons.back()->GetFileName()<<std::endl;
+           if(_debug>0) std::cout<<"CRV sector "<<i<<" ("<<_CRVSectors[i]<<") uses "<<_makeCrvPhotons.back()->GetFileName()<<std::endl;
            break;
         }
       }
       if(tableLoaded) continue;
 
       _makeCrvPhotons.emplace_back(boost::shared_ptr<mu2eCrv::MakeCrvPhotons>(new mu2eCrv::MakeCrvPhotons(_randFlat, _randGaussQ, _randPoissonQ)));
       boost::shared_ptr<mu2eCrv::MakeCrvPhotons> &photonMaker=_makeCrvPhotons.back();
-      photonMaker->LoadLookupTable(_resolveFullPath(_lookupTableFileNames[i]));
+      std::string filespec = _resolveFullPath(_lookupTableFileNames[i]);
+      filedirs.insert( std::filesystem::path(filespec).parent_path() );
+      photonMaker->LoadLookupTable(filespec,_debug);
       photonMaker->SetScintillationYield(_scintillationYields[i]);
-      std::cout<<"CRV sector "<<i<<" ("<<_CRVSectors[i]<<") uses "<<_makeCrvPhotons.back()->GetFileName()<<" with scintillation yield of "<<_scintillationYields[i]<<" photons/MeV"<<std::endl;
+      if(_debug>0) std::cout<<"CRV sector "<<i<<" ("<<_CRVSectors[i]<<") uses "<<_makeCrvPhotons.back()->GetFileName()<<" with scintillation yield of "<<_scintillationYields[i]<<" photons/MeV"<<std::endl;
```

Comment:
The sector-by-sector print outs of the scintillation yields is of some importance, because we are running simulations with different scintillation yields where the log files may be useful to recover the values that were used. What is the requirement to print out information at debug level 0?

Reply:
My goal is to keep asking if prints are necessary and apply some pressure to keep it down.  I'll leave it 
to the group to determine "necessary".   For this study, you could turn the debug to 1 in your fcl.
Or will the changes be made with different light files in different directories, in which 
case the directory path print captures it?  We also have the choice to set the debug level in 
different ways, for example, leave it off by default but turn it on by default for all Production fcl.  
We already do that for other flags.  If the decision is to restore the print, we could try to reduce its
visual impact a bit.

Reply:
@ehrlich-uva  are you happy with Ray's reply or do you still want a change?  If you are happy, please approve the PR.

Reply:
The scintillation yields of the individual CRV sectors is important information. That's why I was wondering whether it could be kept in the log file regardless of the debug level. 
As an alternative, we could set the default debug level to 1 for the CrvPhotonGenerator so that all relevant information gets printed out by default. In this case, debug>0 in MakeCrvPhotons.cc could be changed to debug>1, since this part is less relevant.
@rlcee The lookup-table files and directories don't change when different scintillation yields are used.